Gemfile

```diff
@@ -68,6 +70,7 @@ group :development do
 
   # Speed up commands on slow machines / big apps [https://github.com/rails/spring]
   # gem 'spring'
+  gem 'bullet'
```

Comment:
- [ ] @Team I can see you have installed the bullet gem which can be used to fix N+1 problem, however it's against the project requirement to use it, I would recommend you remove it, as you have already added code to fix the problem
![image](https://github.com/over-geek/blogapp/assets/61408860/3b33a625-d206-48fa-8212-2366adf3dd5d)